up42/asset.py

```diff
@@ -100,9 +70,39 @@ def stac_items(self) -> pystac.ItemCollection:
                 ],
             },
         }
-        pystac_search = pystac_client_aux.search(filter=stac_search_parameters)
-        resulting_item = pystac_search.get_all_items()
-        return resulting_item
+        pystac_asset_search = pystac_client_aux.search(filter=stac_search_parameters)
+        return (pystac_client_aux, pystac_asset_search)
+
+    @property
+    def stac_info(self) -> Optional[pystac.Collection]:
+        """
+        Gets the storage STAC information for the asset as a FeatureCollection.
+        One asset can contain multiple STAC items (e.g. the pan- and multispectral images).
+        """
+        pystac_client_aux, pystac_asset_search = self._stac_search
+        resulting_item = next(
+            pystac_asset_search.get_item_collections(), None
+        )  # up42:asset_id unique, we expect only one results
+        if resulting_item is None:
+            raise ValueError(
+                f"No STAC metadata information available for this asset {self.asset_id}"
+            )
+        collection_id = resulting_item[0].collection_id
+        return pystac_client_aux.get_collection(collection_id)
+
+    @property
+    def stac_items(self) -> pystac.ItemCollection:
+        """ "
+        returns the stac items from an UP42 asset STAC representation.
+        """
+        try:
+            _, pystac_asset_search = self._stac_search
+            resulting_item = pystac_asset_search.get_all_items()
```

Comment:
```suggestion
            resulting_items = pystac_asset_search.get_all_items()
```
nitpick: in case it results in more than one item :)